hack/make/binary

```diff
@@ -62,3 +62,18 @@ echo "Created binary: $DEST/$BINARY_FULLNAME"
 ln -sf "$BINARY_FULLNAME" "$DEST/docker$BINARY_EXTENSION"
 
 hash_files "$DEST/$BINARY_FULLNAME"
+
+# Add nested executables to bundle dir so we have complete set of
+# them available, but only if the native OS/ARCH is the same as the
+# OS/ARCH of the build target
+if [ "$(go env GOOS)/$(go env GOARCH)" == "$(go env GOHOSTOS)/$(go env GOHOSTARCH)" ]; then
+	echo "Copying nested executables into $DEST"
+	(set -x
+	if [ -x /usr/local/bin/runc ]; then
+		cp /usr/local/bin/runc $DEST/
```

Comment:
kinda blows they all go in the bundles/version/binary/folder also i think they will get duped in bundles/version/dynbinary depending on what you are building, since dynbinary calls binary https://github.com/docker/docker/blob/master/hack/make/dynbinary#L9
it lowers the nice organization we have going on but idk if its that big a problem